src/src/Helper/ShellHelper.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace App\Helper;
+
+use App\Entity\Option;
+use App\Factory\TaskFactory;
+use App\Repository\OptionRepository;
+use Doctrine\ORM\EntityManagerInterface;
+
+class ShellHelper
+{
+    private OptionRepository $optionRepository;
+    private EntityManagerInterface $entityManager;
+    private TaskFactory $taskFactory;
+
+    public function __construct(
+        OptionRepository $optionRepository,
+        EntityManagerInterface $entityManager,
+        TaskFactory $taskFactory
+    ) {
+        $this->optionRepository = $optionRepository;
+        $this->entityManager = $entityManager;
+        $this->taskFactory = $taskFactory;
+    }
+
+    private function setOptionValue(string $name, string $value): void
```

Comment:
is this being used?

Reply:
Maybe not :P Just copied it from another class as I was expecting to need it... But I guess we don't need it! Will double check